projects/ui-server/public/app.js

```diff
@@ -14,6 +14,19 @@ const favorites = {
   remove(soundName) {
     this.list = this.list.filter(i => i !== soundName);
   },
+  toggleBtnAsFav(favStar) {
+    const star = favStar;
+    if (star.classList.contains('fav-set')) {
+      star.innerHTML = 'star_outline';
+      this.remove(star.parentElement.dataset.soundName);
+    } else {
+      star.innerHTML = 'star';
+      this.list.push(star.parentElement.dataset.soundName);
+    }
+    star.classList.toggle('fav-set');
+    star.parentElement.classList.toggle('fav');
+    this.save();
```

Comment:
`star` doesn't really need to exist, it's just an alias for `favStar` here.

```suggestion
    if (favStar.classList.contains('fav-set')) {
      favStar.innerHTML = 'star_outline';
      this.remove(favStar.parentElement.dataset.soundName);
    } else {
      favStar.innerHTML = 'star';
      this.list.push(favStar.parentElement.dataset.soundName);
    }
    favStar.classList.toggle('fav-set');
    favStar.parentElement.classList.toggle('fav');
    this.save();
```

Reply:
it's because of `eslint(no-param-reassign)`

Reply:
ech.